core-common/src/main/java/eu/arrowhead/common/CoreCommonConstants.java

```diff
@@ -188,6 +190,16 @@
 	public static final String $QOS_ENABLED_RELAY_TASK_WD =  "${" + QOS_ENABLED_RELAY_TASK + ":" + CoreDefaults.DEFAULT_QOS_ENABLED_RELAY_TASK + "}";
 	public static final String QOS_IS_GATEKEEPER_PRESENT = "gatekeeper_is_present";
 	public static final String $QOS_IS_GATEKEEPER_PRESENT_WD = "${" + QOS_IS_GATEKEEPER_PRESENT + ":" + CoreDefaults.DEFAULT_QOS_IS_GATEKEEPER_PRESENT + "}";
+	public static final String QOS_MONITOR_PROVIDER_TYPE = "monitor_provider_type";
+	public static final String $QOS_MONITOR_PROVIDER_TYPE_WD = "${" + QOS_MONITOR_PROVIDER_TYPE + ":" + CoreDefaults.DEFAULT_QOS_MONITOR_PROVIDER_TYPE + "}";
+	public static final String QOS_MONITOR_PROVIDER_ADDRESS = "monitor_provider_address";
+	public static final String $QOS_MONITOR_PROVIDER_ADDRESS_WD = "${" + QOS_MONITOR_PROVIDER_ADDRESS + ":" + CoreDefaults.DEFAULT_QOS_MONITOR_PROVIDER_ADDRESS + "}";
+	public static final String QOS_MONITOR_PROVIDER_PORT = "monitor_provider_port";
+	public static final String $QOS_MONITOR_PROVIDER_PORT_WD = "${" + QOS_MONITOR_PROVIDER_PORT + ":" + CoreDefaults.DEFAULT_QOS_MONITOR_PROVIDER_PORT + "}";
+	public static final String QOS_MONITOR_PROVIDER_PATH = "monitor_provider_path";
+	public static final String $QOS_MONITOR_PROVIDER_PATH_WD = "${" + QOS_MONITOR_PROVIDER_PATH + ":" + CoreDefaults.DEFAULT_QOS_MONITOR_PROVIDER_PATH + "}";
+	public static final String QOS_MONITOR_PROVIDER_SCHEME = "monitor_provider_scheme";
+	public static final String $QOS_MONITOR_PROVIDER_SCHEME_WD = "${" + QOS_MONITOR_PROVIDER_SCHEME + ":" + CoreDefaults.DEFAULT_QOS_MONITOR_PROVIDER_SCHEME + "}";
```

Comment:
because we only support http or https as scheme, maybe a simple flag (monitor_provider_secure) would be better